pkg/operator/operator.go

```diff
@@ -105,9 +105,11 @@ type Config struct {
 	HiveUseTLS bool
 	HiveCAFile string
 
-	PrestoHost   string
-	PrestoUseTLS bool
-	PrestoCAFile string
+	PrestoHost       string
+	PrestoUseTLS     bool
+	PrestoUseAuth    bool
```

Comment:
Perhaps `PrestoUseClientCertAuth` since there's more than 1 form of auth, and we're using client certs specifically.